lib/archethic/bootstrap/network_init.ex

```diff
@@ -31,13 +32,19 @@ defmodule ArchEthic.Bootstrap.NetworkInit do
   alias ArchEthic.TransactionChain.TransactionData.Ledger
   alias ArchEthic.TransactionChain.TransactionData.UCOLedger
   alias ArchEthic.TransactionChain.TransactionData.UCOLedger.Transfer
+  alias ArchEthic.TransactionChain.TransactionData.Ownership
 
   alias ArchEthic.TransactionChain.TransactionSummary
 
   require Logger
 
   @genesis_seed Application.compile_env(:archethic, [__MODULE__, :genesis_seed])
 
+  @genesis_origin_public_keys Application.compile_env!(
+                                :archethic,
+                                [NetworkInit, :genesis_origin_public_keys]
```

Comment:
I don't think the alias is necessary, you can use `__MODULE__` if you want to mention the current module.

Reply:
updated alias and added formatter for origin_shared_secrets content in explorer